ios-xcframework/build.sh

```diff
@@ -18,13 +18,130 @@ function _xcodebuild {
     fi
 }
 
+function log {
+  if [[ $# -eq 1 ]]; then
+    PREFIX=''
+    MESSAGE=$1
+  else
+    PREFIX=":$1: "
+    MESSAGE=$2
+  fi
+
+  set +u
+  if [[ -n "$BUILDKITE" ]]; then
+    echo "--- $PREFIX$MESSAGE"
+  else
+    echo "$MESSAGE"
+  fi
+  set -u
+}
+
 DERIVED_DATA_PATH=./DerivedData
 DESTINATION='generic/platform=iOS'
 
 rm -rf $DERIVED_DATA_PATH
 
+MAIN_FRAMEWORK_NAME=Gutenberg
+WORKSPACE="./XCFrameworkScaffold.xcworkspace"
+SCHEME=$MAIN_FRAMEWORK_NAME
+
+log "Building $SCHEME..."
+
 _xcodebuild clean build \
-  -workspace ./XCFrameworkScaffold.xcworkspace \
-  -scheme 'Gutenberg' \
+  -workspace $WORKSPACE \
+  -scheme $SCHEME \
   -destination "$DESTINATION" \
   -derivedDataPath "$DERIVED_DATA_PATH"
```

Comment:
Do we need this additional build, considering there are two build command below for iOS and Simulator platforms?

Reply:
Good point. I used this as a first step to verify the build, but it's redundant now.

Addressed in 6967ed81.